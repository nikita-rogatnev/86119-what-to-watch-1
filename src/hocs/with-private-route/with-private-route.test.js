import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import withPrivateRoute from "./with-private-route";

Enzyme.configure({adapter: new Adapter()});

it(`withPrivateRoute renders correctly`, () => {
  const tree = shallow(<withPrivateRoute/>);

  expect(toJson(tree)).toMatchSnapshot();
});
