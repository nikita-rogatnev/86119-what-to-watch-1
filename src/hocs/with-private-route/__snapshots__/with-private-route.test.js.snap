// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPrivateRoute renders correctly 1`] = `<withPrivateRoute />`;
